CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
+Version 0.4.1 (05/07/2018)
+
+- Add support for JSONC
+- Improved labeling in JSON outline, similar to Visual Studio
+  - Use displayName tag for label of resources when available
+  - Shorten labels when possible (e.g. parameters['abc'] -> &lt;abc&gt;)
+- Engineering improvements (tests, lint, telemetry)
```

Comment:
Do I add this to the cosmosdb extension too? This would also apply there...

Reply:
A lot more happened in ARM.